src/legacy/ui/public/vis/editors/default/components/agg_group_helper.tsx

```diff
@@ -17,22 +17,26 @@
  * under the License.
  */
 
-import { findIndex, reduce, isEmpty } from 'lodash';
+import { findIndex, isEmpty } from 'lodash';
 import { AggConfig } from '../../../../agg_types/agg_config';
 import { AggsState } from './agg_group_state';
 
 const isAggRemovable = (agg: AggConfig, group: AggConfig[]) => {
-  const metricCount = reduce(
-    group,
-    (count, aggregation: AggConfig) => {
-      return aggregation.schema.name === agg.schema.name ? ++count : count;
-    },
-    0
-  );
+  const metricCount = group.reduce((count, aggregation: AggConfig) => {
+    return aggregation.schema.name === agg.schema.name ? ++count : count;
```

Comment:
Why do we need to modify count here?

Reply:
This code calculates how many aggs there are with the same schema name as the current agg has.